### DISCUSSION TITLE
Is a window the same as a split? (and related)

### DISCUSSION THREAD

**Discussion by jerabaul29** [helix-editor/helix#8726] [General]
I get a bit confused around the concepts of window vs split. For example, after doing a ctrl-w , I get:

![Screenshot from 2023-11-04 20-29-48](https://github.com/helix-editor/helix/assets/8382834/f417a96e-1463-4af9-8ddd-e1ae2bce36e4)

which mentions both windows and splits. Are the 2 concepts the same or not? For me it looks like they are the same (for example, if I open a split, then using close window closes the new split; similarly, next window moves to the next split).

If window = split, maybe it would be useful to default to one single word? (I would vote for only using the word split not the word window, but the opposite would certainly be fine too).

If window != split, any way this can be explained / documented? Wondering if having a short introduction to the concepts of buffers, windows / splits, maybe at some point tab if something along the lines of https://github.com/helix-editor/helix/pull/7109 gets merged, could be useful for new users? :)

**Comment by the-mikedavis** [comment]
Essentially they're they same but the wording is actually a little precise here: all splits are windows but if you only have one window open it's not really a "split." I think it would be fine to just call them all windows though for simplicity.

(Internally "windows" are called `View`s - it's similar to how we call `Document`s "buffers.")
